src/constants/date.constants.ts

```diff
@@ -6,4 +6,8 @@ export const DAY_MAPPING = {
   fri: "금요웹툰",
   sat: "토요웹툰",
   sun: "일요웹툰",
-} as const;
\ No newline at end of file
+} as const;
+
+export type DayOfWeek = keyof typeof DAY_MAPPING;
+
+export const DAYS: readonly DayOfWeek[] = ["sun","mon","tue","wed","thu","fri","sat"];
```

Comment:
```suggestion
export const DAYS = ["sun", "mon", "tue", "wed", "thu", "fri", "sat"] as const;
```
`as const` 활용을 추천드립니다.